db/schema_tables.cc

```diff
@@ -1736,6 +1742,14 @@ mutation make_scylla_tables_mutation(schema_ptr table, api::timestamp_type times
     // In-memory tables are deprecated since scylla-2024.1.0
     // FIXME: delete the column when there's no live version supporting it anymore.
     // Writing it here breaks upgrade rollback to versions that do not support the in_memory schema_feature
+    if (auto tablet_hints_cdef = scylla_tables()->get_column_definition("tablet_hints")) {
+        if (table->has_tablet_hints()) {
+            auto& map = table->raw_tablet_hints();
+            m.set_clustered_cell(ckey, *tablet_hints_cdef, make_map_mutation(map, *tablet_hints_cdef, timestamp));
+        } else {
+            m.set_clustered_cell(ckey, *tablet_hints_cdef, atomic_cell::make_dead(timestamp, gc_clock::now()));
+        }
+    }
```

Comment:
What if the column isn't there?

Reply:
This is done inside `if (auto tablet_hints_cdef = scylla_tables()->get_column_definition("tablet_hints"))` so the column should be there as far as the schema goes, or did you mean something else?

Reply:
Ah, I'm used to checking the schema feature, not to checking the schema. It's okay.